src/test/analyzer_test.ts

```diff
@@ -42,7 +44,7 @@ suite('Analyzer', () => {
 
     test('fragment to deps list has only uniques', () => {
       const config = new ProjectConfig({
-        root: `test/static/analyzer-data`,
+        root: `src/test/static/analyzer-data`,
```

Comment:
This feels weird to me now because the static test assets are copied into `lib` but the test is loading them from `src`.

Fix my cognitive dissonance please!